.github/workflows/import-nixpkgs.yml

```diff
@@ -17,14 +17,18 @@ jobs:
         channel:
           - unstable
           - 21.11
-          - 21.05
 
     env:
       RUST_LOG: debug
       FI_ES_EXISTS_STRATEGY: abort
       FI_ES_URL: ${{ secrets.ELASTICSEARCH_URL }}
 
     steps:
+    - name: Increase swap space
+      uses: pierotofy/set-swap-space@v1.0
+      with:
+        swap-size-gb: 10
+
```

Comment:
Is this necessary due to this pr or could it be a seperate pr itself?

Reply:
Yes, this is necessary due to the extra evaluation caused by `--out-path`